cnv/cnv_attaching_resource_multiple_networks.adoc

```diff
@@ -0,0 +1,28 @@
+[id="attaching-to-multiple-networks"]
+= Attaching a resource to multiple networks
```

Comment:
I find the term "resource" too generic and vague. Would you replace it with "virtual machine"?

Reply:
Yes! That's what I originally had, but since you can also attach pods and VMIs to multiple networks, I wasn't sure if it was accurate enough. Probably better to be too specific than too vague, though.